docs/src/components/layouts/utility-page-layout/utility-info-table.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as utilityModules from '../../../data/utility-modules.json';
+
+const renderStateModifiers = (activeStates) => {
+	// TODO: Make these links to the how to use utility page.
```

Comment:
Do the TODO here please. (I see you added this to the class listings you made)

Reply:
So a couple of things here. 
1. The links should be an anchor to the section of that page. So not just to the utility page. 
2. remove the todo if you have todone it...